src/ZipFile.jl

```diff
@@ -141,45 +141,78 @@ type WritableFile <: IO
 	end
 end
 
-type Writer
-	_io :: IO
-	files :: Vector{WritableFile} # files (being) written
-	_current :: Union(WritableFile, Nothing)
-	_closed :: Bool
+if VERSION > v"0.3.11" then
```

Comment:
there will likely be at least one more 0.3 release, so not good to hardcode the patch number here.